### DISCUSSION TITLE
注册服务后无法启动服务

### DISCUSSION THREAD

**Discussion by zxa3256855565** [1715173329/UnblockNeteaseMusic#701] [Q&A]
![image](https://user-images.githubusercontent.com/75650810/169497574-e1f1a083-003d-451b-b0cf-a2e306e26073.png)
如图示，
时间为2022.5.20
系统版本如下：
![image](https://user-images.githubusercontent.com/75650810/169497683-cfa9fe6f-50ef-45f3-b420-723969511549.png)

**Comment by pan93412** [comment]
直接執行「可執行文件的路徑」裡面的檔案，顯示什麼？

**Reply by zxa3256855565** [reply]
![image](https://user-images.githubusercontent.com/75650810/169503291-838655fe-e167-4686-a0ae-d744b9eec2a1.png)

**Comment by 1715173329** [comment]
#620 或许需要先降级下？

**Reply by zxa3256855565** [reply]
谢谢  已根据#620解决了
